clang/include/clang/Basic/DiagnosticCommonKinds.td

```diff
@@ -349,6 +349,8 @@ def warn_invalid_feature_combination : Warning<
 def warn_target_unrecognized_env : Warning<
   "mismatch between architecture and environment in target triple '%0'; did you mean '%1'?">,
   InGroup<InvalidCommandLineArgument>;
+def warn_knl_knm_target_supports_remove : Warning<
```

Comment:
supports -> support remove->removed to match the string

Reply:
Thanks review, addressed in 3eb6b009b14202d4c14bdc691d80550bac4c9aa7